### ISSUE TITLE
Unable to use getTokenBalances

### ISSUE THREAD

**Issue by Jake-Mulhern** [alchemyplatform/alchemy-sdk-js#172]
### [REQUIRED] Environment

- Alchemy SDK version: 2.2.0

### [REQUIRED] Describe the problem:
I have a react-native project where we are building a wallet.

I am attempting to get all erc20 balances in a users wallet using the following line of code:
`const balances = await alchemy.core.getTokenBalances(address, 'erc20')`

When this code runs, however, I get the following error:

`"invalid 2nd argument: contract_addresses was not a valid contract address array, string literals 'DEFAULT_TOKENS' or 'erc20', or a valid options object.\"`

I've tried using 'DEFAULT_TOKENS' as well and get the same error.

#### How to reproduce:

Set up a react-native project and attempt to get all token balances using the following line of code:

const balances = await alchemy.core.getTokenBalances(address, 'erc20')

Run the project with a valid wallet address and observe the error.

#### Relevant code or sample repro:

```
import 'react-native-get-random-values';
import '@ethersproject/shims';

import { Network, Alchemy, Wallet } from 'alchemy-sdk';
import {ETHERS_NETWORK, ALCHEMY_API_KEY} from '@env';

const settings = {
    apiKey: ALCHEMY_API_KEY,
    network: Network.ETH_GOERLI
};
const alchemy = new Alchemy(settings);

export const getAllBalances = async (address) => {
    try {
        const balances = await alchemy.core.getTokenBalances(address, 'erc20');
        return balances;
    } catch (err) {
        console.log(err.message);
    }
}

```

**Comment by thebrianchen** [comment]
@Jake-Mulhern thanks for filing an issue. To specify the category, you'll need to pass in the `erc20` category into the options object (please refer to the [docs](https://github.com/alchemyplatform/alchemy-sdk-js/blob/main/docs-md/classes/CoreNamespace.md#gettokenbalances)).

So to clarify, the request should be:
```js
await alchemy.core.getTokenBalances(address, {type: TokenBalanceType.ERC20});
```

I'm going to close this issue since it's working as intended, but feel free to ask any other questions you may have!

**Comment by Jake-Mulhern** [comment]
That seems to work.  Thank you for the response.  I do have a quick follow up question and am not sure where best to ask it so I'll post it here @thebrianchen 

I have been using the goerli faucet to get eth in my test wallet however it returns an empty array.  Do I need to configure anything else to get the eth balance in my account?  When I set up the app through alchemy and got my API key I made sure to set up the app for the Goerli network.
